modules/building/pages/creating.adoc

```diff
@@ -91,6 +91,126 @@ For further details on PipelineRun permissions, please refer to the https://pipe
 
 . Once the PR is made, a build pipeline should start. You can track its progress in the {ProductName} UI or you can see the final status in GitHub after the pipeline completes. If the pipeline is successful, you can merge the PR.
 
+== Using different from default image repository
+
+When using ImageRepository object, you will end up with image repository in
+`quay.io/redhat-user-workloads/workspace-tenant/`, if you want to use your own repository
```

Comment:
This isn't strictly true. This is where we have configured our controller to push content.  Can you just generalize this since we are in the upstream docs?

It doesn't matter where the repo is. What matters is that it is a repository of the build platform's choosing and access will be given to your SAs.